test/utils.test.ts

```diff
@@ -1101,4 +1103,81 @@ describe('utils', () => {
 
   })
 
+
+  describe('flattenSha256() ', () => {
+
+    it('flattern data', () => {
+
+
+      expect(flattenSha256(1))
+        .to.be.equal("4bf5122f344554c53bde2ebb8cd2b7e3d1600ad631c385a5d7cce23c7785459a");
```

Comment:
How do we know these hashed values r correct? Combine them w/ .scrypt to compare what Typescript calculates w/ sCrypt.